### PR TITLE
Add Leptos / web frameworks

### PR DESCRIPTION
**Pull request by Skyliegirl33**
## Website
https://leptos.dev/

## Repo
https://github.com/leptos-rs/leptos

## Examples
https://leptos.dev/
https://benwis-leptos.fly.dev/
https://viz.rs/
https://sadgpt.netlify.app/
https://seanaye-leptos-deno.deno.dev/

Unable to detect these ones, aside from `<!---->` lines, which felt too generic to match.
https://isotoxaldev.github.io/le_snake/
https://play.aiken-lang.org/ (https://github.com/aiken-lang/play)
https://levuminhhuy.vercel.app/

### CODE REVIEW

File: src/technologies/l.json [hunk 1]
```diff
@@ -536,6 +536,23 @@
     "oss": true,
     "website": "https://lenis.studiofreight.com"
   },
+  "Leptos": {
+    "cats": [
+      18
+    ],
+    "description": "Leptos is a full-stack, isomorphic Rust web framework leveraging fine-grained reactivity to build declarative user interfaces.",
+    "html": "<!--hk=_.*",
```
Comment (nurbek91) [inline src/technologies/l.json:544]: I could not find this trace in the examples

File: src/technologies/l.json [hunk 2]
```diff
@@ -536,6 +536,23 @@
...
+    "icon": "Leptos.svg",
+    "implies": "Rust",
+    "js": {
+      "__LEPTOS_PENDING_RESOURCES": "",
+      "__LEPTOS_RESOLVED_RESOURCES": "",
+      "__LEPTOS_RESOURCE_RESOLVERS": ""
+    },
+    "oss": true,
+    "scripts": "export function microtask\\(f\\)\\;confidence:75",
```
Comment (nurbek91) [inline src/technologies/l.json:553]: I could not find this trace in the examples also 
It is better to remove them, they are resource intensive.
Usually they are added if there is no other possibility or if there are enough examples where they are used.

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks for PR!!!

File: src/technologies/l.json (hunk 2 above)
Comment (Skyliegirl33) [reply src/technologies/l.json:553]: This is certainly in the examples, it's a fairly unique identifier for Leptos so that's why I added it.

image

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**